packages/components/src/tree-grid/stories/index.story.tsx

```diff
@@ -16,7 +16,8 @@ import { Button } from '../../button';
 import InputControl from '../../input-control';
 
 const meta: Meta< typeof TreeGrid > = {
-	title: 'Components (Experimental)/TreeGrid',
+	title: 'Components (Experimental)/Navigation/TreeGrid',
+	id: 'components-treegrid',
```

Comment:
As this component is still formally experimental, if we change its ID, it will cause the previous URL not to work:

http://localhost:50240/?path=/docs/components-experimental-treegrid--docs

To fix that, we have 2 options:
- Use `id: 'components-experimental-treegrid'`
- Add the component to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`. 

Since the component is still experimental, I'd prefer the former option. Once we stabilize the component, we can always add it to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`